app/windows/winget/templates/Headlamp.Headlamp.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+PackageIdentifier: Headlamp.Headlamp
+PackageVersion: __PACKAGE_VERSION__
+PackageLocale: en-US
+Publisher: Headlamp
+PublisherUrl: https://headlamp.dev
+PackageName: Headlamp
+License: Apache 2.0
+LicenseUrl: https://github.com/headlamp-k8s/headlamp/blob/main/LICENSE
+ShortDescription: Kubernetes web UI.
+Description: Headlamp was created to be a Kubernetes web UI that has the traditional functionality of other web UIs/dashboards available (i.e. to list and view resources) as well as other features.

```

Comment:
I know this was here before. I just wonder now if we should have something clearer, like we have in Github: `A Kubernetes web UI that is fully-featured, user-friendly and extensible`

Reply:
Added!